test-frame-common/src/main/java/io/skodjob/testframe/TestFrameConstants.java

```diff
@@ -45,7 +45,7 @@ public interface TestFrameConstants {
     /**
      * Global timeout in milliseconds.
      */
-    long GLOBAL_TIMEOUT_SHORT = Duration.ofMinutes(5).toMillis();
+    long GLOBAL_TIMEOUT_SHORT = Duration.ofMinutes(3).toMillis();
```

Comment:
Change the constant for the default resource timeout in the KubeResourceManager please. Use the MEDIUM one there.